### PR TITLE
[cli] Credentials: support platform flag

### PR DESCRIPTION
**Pull request by Simek**
# Checklist

- [X] I've added an entry to [CHANGELOG.md](https://github.com/expo/eas-cli/blob/main/CHANGELOG.md#main) if necessary. You can comment this pull request with `/changelog-entry [breaking-change|new-feature|bug-fix|chore] [message]` and CHANGELOG.md will be updated automatically.

# Why

Fixes ENG-5974

# How

Add jest command flags in the similar way it has been done in other commands. Additionally, I have used `selectPlatformAsync` instead of manually defined prompt.

# Test Plan

After the change I have built and tested command locally via `yarn eas`.

## Preview

Screenshot 2022-08-14 133630

Screenshot 2022-08-14 134023

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,8 @@ This is the log of notable changes to EAS CLI and related packages.
 
 ### 🎉 New features
 
+- Add support for passing platform flag to `credentials` command.
```
Comment (dsokal) [inline CHANGELOG.md:11]:
```suggestion
- Add support for passing platform flag to `credentials` command. ([#1277](https://github.com/expo/eas-cli/pull/1277) by [@Simek](https://github.com/Simek))
```

File: packages/eas-cli/src/credentials/manager/SelectPlatform.ts [hunk 2]
```diff
@@ -1,22 +1,20 @@
...
+import { selectPlatformAsync } from '../../platform';
 import { ManageAndroid } from './ManageAndroid';
 import { ManageIos } from './ManageIos';
 
 export class SelectPlatform {
+  private readonly flagPlatform: string;
+
+  constructor(platform: string) {
+    this.flagPlatform = platform;
+  }
```
Comment (dsokal) [inline packages/eas-cli/src/credentials/manager/SelectPlatform.ts:10]: One-liner:
```suggestion
  constructor(private readonly platformFlag: string) {}
```

**Review by dsokal — APPROVED** [review]
🎉

File: packages/eas-cli/src/credentials/manager/SelectPlatform.ts (hunk 2 above)
Comment (wkozyra95) [reply packages/eas-cli/src/credentials/manager/SelectPlatform.ts:10]: flagPlatform should be optional

**Review by wkozyra95 — APPROVED** [review]

File: packages/eas-cli/src/credentials/manager/SelectPlatform.ts [hunk 3]
```diff
@@ -1,22 +1,18 @@
-import { promptAsync } from '../../prompts';
+import { selectPlatformAsync } from '../../platform';
 import { ManageAndroid } from './ManageAndroid';
 import { ManageIos } from './ManageIos';
 
 export class SelectPlatform {
+  constructor(private readonly flagPlatform?: string) {
+    this.flagPlatform = flagPlatform;
```
Comment (dsokal) [inline packages/eas-cli/src/credentials/manager/SelectPlatform.ts:7]: This is redundant
```suggestion
```

**Review by dsokal — APPROVED** [review]

**Pull request merged**